src/index.html

```diff
@@ -9,6 +12,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo-link">
```

Comment:
I think it would be more useful to include `<img />` tag in link - that's what it's for, instead of setting element's background-image 😄

Reply:
done in solution2 it's <div><a><img></a>/</div>